### DISCUSSION TITLE
Using a footer in README.MD

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21911] [New to GitHub]
<p>I am attempting to create a footer in my README.MD file which should have three sections that display horizontally across the bottom of the document. This works in my markdown editor preview in Visual Studio 2017 but unfortunately Github insists in displaying the sections vertically. The Html I used follows here:</p>
<pre><code>&lt;footer&gt;
&lt;p style="float:left; width: 20%;"&gt;
Copyright © Jonathan Gossage, 2018
&lt;/p&gt;
&lt;p style="float:left; width: 60%; text-align:center;"&gt;
&lt;a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"&gt;&lt;img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /&gt;&lt;/a&gt;&lt;br /&gt;This work is licensed under a &lt;a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"&gt;Creative Commons Attribution-ShareAlike 4.0 International License&lt;/a&gt;
&lt;/p&gt;
&lt;p style="float:left; width: 20%;"&gt;
Email addresses will be added when they are setup.
&lt;/p&gt;
&lt;/footer&gt;
</code></pre>
<p>I have tried using both &lt;div&gt; blocks and &lt;p&gt; without success. Can anyone suggest how to get these sections displayed horizontally on Github?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>GitHub sanitizes the HTML that can be generated from the Markdown in documents to prevent security holes from being exploited. I don’t know of a way to get those things displayed horizontally in the way that you’re describing other than <a href="https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown" rel="nofollow noopener">using a Markdown table</a>.</p>
